/**:
  ros__parameters:
    fixed_robot_body: false
    